app/controllers/users_controller.rb

```diff
@@ -1,8 +1,16 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: %i(index update edit destroy)
```

Comment:
nên dùng except để gọn hơn, mà show thì cũng phải check login chứ em?
```suggestion
  before_action :logged_in_user, except: %i(new create show)
```

Reply:
dạ tại em thấy cái show chỉ như là xem thông tin của nhau thôi ạ, nên cũng không cần check log in.